From 04f4c5d8f782ace624cedc64e2974ea7f60f74de Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@turris.com>
Date: Thu, 18 Jun 2026 16:03:00 +0200
Subject: [PATCH] ifstat: Add openssl dependency

Fixes:
```
Package ifstat is missing dependencies for the following libraries:
libcrypto.so.3
libssl.so.3
```
---
 net/ifstat/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/ifstat/Makefile b/net/ifstat/Makefile
index ddbef5dbe..9ebc8093e 100644
--- a/net/ifstat/Makefile
+++ b/net/ifstat/Makefile
@@ -26,7 +26,7 @@ define Package/ifstat
   CATEGORY:=Network
   TITLE:=InterFace STATistics Monitoring
   URL:=http://gael.roualland.free.fr/ifstat/
-  DEPENDS:=+IFSTAT_SNMP:libnetsnmp
+  DEPENDS:=+IFSTAT_SNMP:libnetsnmp +libopenssl
 endef
 
 define Package/ifstat/description
-- 
2.54.0

